### PR TITLE
Add basic UI design

### PR DESCRIPTION
**Pull request by jchernandez87**
### Changes: 😃 

- Add a components folder.
- Add Calculator.js and Calculator.css to my components folder.
- Add basic styling to my component using CSS grid.
- Use a class component Calculator to make the basic HTML using jsx.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,11 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Math Magicians
-        </a>
-      </header>
+      <Calculator />
     </div>
   );
```
Comment (arslanbisharat) [inline src/App.js:10]: - [ ] App.js should have a class-based component so please convert this piece of code to a class-based component.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,21 @@
-import logo from './logo.svg';
+/* eslint-disable class-methods-use-this */
```
Comment (arslanbisharat) [inline src/App.js:1]: - [ ] As requested in the previous review we should avoid using disable eslint. 
In your case, you should use this.state to add proper functionalities for future milestones. 

**Here is a price of code that might help you**

```
class Calculator extends React.Component {
  constructor(props) {
    super(props);
    this.state = {
      total: null,
      next: null,
      operation: null,
    };
    this.handleClick = this.handleClick.bind(this);
  }
```

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/App.js (hunk 2 above)
Comment (jchernandez87) [reply src/App.js]: The problem was that I installed the javascript linters and not the ones for react.

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @jchernandez87, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**